dashboard/app/helpers/levels_helper.rb

```diff
@@ -255,7 +255,8 @@ def app_options
       view_options(
         lesson_position: @script_level.lesson.absolute_position,
         level_position: @script_level.position,
-        next_level_url: @script_level.next_level_or_redirect_path_for_user(current_user, @lesson)
+        next_level_url: @script_level.next_level_or_redirect_path_for_user(current_user, @lesson),
+        current_level_url: @script_level.path,
```

Comment:
Because this is only defined for script levels, I'm wondering if it's better to name this `current_script_level_url`? Or maybe at least just leaving a comment in `view_options_helper` - just want to make sure there isn't confusion later around why there isn't a `current_level_url` for a `/levels/` page

Reply:
Sure - will update.

Reply:
@sanchitmalhotra126 @bethanyaconnor - what do you think of updating `next_level_url` to `next_script_level_url` and `nextLevelUrl` to `nextScriptLevelUrl`?

Reply:
Won't consider including change in this PR but just curious what y'all thought. Thanks!

Reply:
No strong preference - for me, "next level" implies that we're in a level progression, so the fact that it's only present for script levels makes sense to me. I just felt that "current level" was a little more ambiguous since it could apply to both script level and level pages.